Services/DeviceMethods.cs

```diff
@@ -136,7 +139,8 @@ private void MethodExecution(MethodRequest methodRequest)
                 this.scriptInterpreter.Invoke(
                     this.cloudToDeviceMethods[methodRequest.Name],
                     scriptContext,
-                    this.deviceState);
+                    this.deviceState,
```

Comment:
Now that you're passing properties separately, do you want to add them to the log.Debug above?
i.e. this:

this.log.Debug("Executing method for device", () => new
                 {	                 {
                     this.deviceId,	                     this.deviceId,
                     methodName = methodRequest.Name,	                     methodName = methodRequest.Name,
                     this.deviceState	                     this.deviceState
                 });	                 });

Reply:
Fixed